design-docs/img/annotation_processing_javaPluginConfigurations.graphml

```diff
@@ -0,0 +1,363 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
This image is intended to eventually replace `subprojects/docs/src/docs/userguide/img/javaPluginConfigurations.graphml` in the user guide.